lib/components/narrative/default/default-itinerary.js

```diff
@@ -167,6 +213,8 @@ export default class DefaultItinerary extends NarrativeItinerary {
                   options.selection = this.props.sort.type
                 }
                 options.LegIcon = LegIcon
+                options.timeFormat = this.props.use24HourFormat ? 'H:mm' : 'h:mm a'
```

Comment:
Please destructure the `use24HourFormat` prop at the beginning of the `render` method with the rest of the props.

Reply:
Would the same be true for the this.props.sort.type above?

Reply:
Yes please!

Reply:
Addressed in https://github.com/opentripplanner/otp-react-redux/pull/375/commits/5e2b6708003925b1163418bd629f3870889ec6a2.